scripts/run_critical_tests.py

```
@@ -48,15 +47,19 @@ async def test_captcha_solver():
+    success, stdout, stderr = run_command(
+        "python -m src.gengowatcher.main captchasetup --help"
+    )
     if success:
         logger.info("✅ CAPTCHA setup command available")
     else:
         logger.error("❌ CAPTCHA setup command not available")
         return False
 
     # Test 2: Check CAPTCHA test command
-    success, stdout, stderr = run_command("python -m src.gengowatcher.main captchatest --help")
+    success, stdout, stderr = run_command(
```

Ruff (0.14.14)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)